content/oto-morph.md

```diff
@@ -0,0 +1,161 @@
+# Morphological Inflection for Oto-Manguean Languages
```

Comment:
This needs a YAML metadata header for it to build as a web page, i.e., the bit where you list the title and authors. Please see [the example post](https://raw.githubusercontent.com/JonathanReeve/course-multilingual-technologies/master/content/first-post.md).